core/paths/transientfile/create_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+//+build !windows
+
+package transientfile
```

Comment:
Not necessarily. However, note that these are white-box tests since they are designed to exercise specific codepaths in the package implementation (e.g. passing a file vs passing a relative path) and where made internal for this reason. No particular objection to renaming the package though if you think it would be more consistent.